### PR TITLE
added post : parallel lines

### PR DESCRIPTION
**Pull request by reversefourseasons**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by AvikantSrivastava** [comment]
@reversefourseasons Please remove the `assets/custom/.DS_Store` file from the commit

**Comment by KaushikiKrity** [comment]
This is done.

**Comment by KaushikiKrity** [comment]
@AvikantSrivastava bhaiyaa review this

File: Gemfile.lock [hunk 1]
```diff
@@ -76,7 +76,7 @@ DEPENDENCIES
   kramdown-parser-gfm
 
 RUBY VERSION
-   ruby 2.7.0p0
```
Comment (AvikantSrivastava) [inline Gemfile.lock]: Please don't change the ruby version

File: Gemfile.lock [hunk 2]
```diff
@@ -76,7 +76,7 @@ DEPENDENCIES
...
+   ruby 2.6.3p62
 
 BUNDLED WITH
-   2.0.2
```
Comment (AvikantSrivastava) [inline Gemfile.lock]: Don't make changes to `Gemfile.lock`

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
> @AvikantSrivastava bhaiyaa review this

Looks like the bin files are not deleted yet

File: Gemfile.lock (hunk 1 above)
Comment (reversefourseasons) [reply Gemfile.lock]: changed that. please review it again, bhaiya.

File: Gemfile.lock (hunk 2 above)
Comment (reversefourseasons) [reply Gemfile.lock]: can you please share the location of the .DS_Store file. I just deleted two of them.

Comment (reversefourseasons) [reply Gemfile.lock]: found them. deleted them. please review it. thank you.

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
There is one bin file still left `.DS_Store`

**Comment by reversefourseasons** [comment]
> There is one bin file still left `.DS_Store`

removed the file.

**Review by AvikantSrivastava — APPROVED** [review]
Awesome, the changes look great.
Thanks for contributing to [LEO MVIT](https://github.com/leomvit)

**Comment by AvikantSrivastava** [comment]
@kaushikikrityy @reversefourseasons 
Look at the [test deployment link](https://deploy-preview-118--leomvit.netlify.app/), Looks like your blog isnt added

**Comment by KaushikiKrity** [comment]
> @kaushikikrityy @reversefourseasons
> Look at the [test deployment link](https://deploy-preview-118--leomvit.netlify.app/), Looks like your blog isnt added

Just saw!!!! Working on it!

**Comment by reversefourseasons** [comment]
@AvikantSrivastava it worked on the local server. pushed that again. please review.

**Pull request closed without merging**